go.mod

```diff
@@ -136,7 +137,7 @@ replace (
 	github.com/microcosm-cc/bluemonday v1.0.2 => github.com/microcosm-cc/bluemonday v1.0.16
 	github.com/miekg/dns v1.0.14 => github.com/miekg/dns v1.1.25
```

Comment:
You should be able to specify the version that you would like to use above, instead of replacing

Reply:
Same as other CVE comments